### PR TITLE
exceptions.rdoc : Add heads up about ensure not returning implicitly

### PR DESCRIPTION
**Pull request by pimpin**
There is a [weird gotcha](https://dev.to/okuramasafumi/be-sure-ensure-doesn-t-return-value-implicitly-8gp) I already forgot twice.... and regret not to have found in doc before.

### CODE REVIEW

**Review by BurdetteLamar — APPROVED** [review]

**Comment by pimpin** [comment]
Travis CI [is failing](https://app.travis-ci.com/github/ruby/ruby/jobs/618532135) it's likely not related to the PR content.

Do I have to update branch and trigger checks until Travis CI passed, to have the PR merged automatically ? Or shall I just wait for someone to get the PR merged ?
cc @BurdetteLamar @nobu (thanks for help)

**Pull request merged**
